.bazelci/postsubmit.yml

```diff
@@ -1,52 +1,41 @@
 ---
-matrix:
-  bzlmod_platforms: ["centos7_java11_devtoolset10", "ubuntu2004", "macos", "windows"]
 
 tasks:
-  bzlmod_builds:
-    name: "Bazel Bzlmod build"
-    platform: ${{ bzlmod_platforms }}
-    build_flags:
-      - "--config=bzlmod"
-    build_targets:
-      - "//src:bazel"
-    test_targets:
-      - "//src/test/cpp/..."
   centos7_java11_devtoolset10:
     shell_commands:
       - sed -i.bak -e 's/^# android_sdk_repository/android_sdk_repository/' -e 's/^#
-        android_ndk_repository/android_ndk_repository/' WORKSPACE
-      - rm -f WORKSPACE.bak
+        android_ndk_repository/android_ndk_repository/' WORKSPACE.bzlmod
+      - rm -f WORKSPACE.bzlmod.bak
```

Comment:
why are we creating this `.bak` in the first place if we delete it right away?

Reply:
It's just for compatibility with macOS, where `sed` doesn't support in-place replacing without a `.bak` file.

Reply:
I didn't know the original reason, but I guess it's just easier to keep them the same so you can search and edit all places just one time when you need some changes. We can clean this up in another PR if necessary.